lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -1321,9 +1321,10 @@ TypeSystemSwiftTypeRefForExpressions::TypeSystemSwiftTypeRefForExpressions(
 void TypeSystemSwiftTypeRefForExpressions::PerformCompileUnitImports(
     SymbolContext &sc) {
   Status error;
-  lldb::StackFrameWP stack_frame;
+  // FIXME: this is uninitialized!
+  lldb::ProcessSP process_sp;
```

Comment:
is this intended to be fixed before merging?

Reply:
Fix is in https://github.com/apple/llvm-project/pull/4077